docs/development.rst

```diff
@@ -275,7 +296,7 @@ Code Review
 -----------
 
 Contributions to Girder are done via pull requests with a core developer
-accepting a PR by saying it "Looks good to me" or LGTM. At this point, the
+approving the PR with GitHub review system. At this point, the
```

Comment:
I think we should remove the whole section on Code Review since it is redundant with our CONTRIBUTING.md file.